frontend/src/Frontend/Network.hs

```diff
@@ -394,9 +396,12 @@ maxCoinPrecision = 12
 defaultTransactionTTL :: TTLSeconds
 defaultTransactionTTL = TTLSeconds (8 * 60 * 60) -- 8 hours
 
--- | TODO: Better defaults!!!
+-- Taken from https://github.com/kadena-io/chainweb-node/blob/85688ea0182d1b1ab0d8d784a48b4851a950ec7a/src/Chainweb/Chainweb.hs#L344
+chainwebGasLimit :: Num a => a
+chainwebGasLimit = 1e5
```

Comment:
Is the default that I set really the limit? Ouch.

Reply:
Ha, yeah, maybe we should drop the default a little? Unless that requirement came from kadena

Reply:
The limit will change to 1e6 soon (https://github.com/kadena-io/chainweb-node/commit/cf83c0f393825a1958e4c1a4007f90606f867e78#diff-1f9807c50e8c60a1c91ca0f4708cb6d1R347).